youtube_dl/extractor/iqm2.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+from .generic import GenericIE
+
+# IQM2 aka Accela is a municipal meeting management platform that
+# (among other things) stores livestreamed video from municipal
+# meetings.  After a hefty (several-hour) processing time, that video
+# is available in easily downloadable form from their web portal, but
+# prior to that, the video can only be watched in realtime through
+# JWPlayer. This extractor is designed to download the realtime video
+# prior to download links being available. For more info on Accela, see:
+#   http://www.iqm2.com/About/Accela.aspx
+#   http://www.accela.com/
+
+# This processing makes it challenging to produce a test case for,
+# because the extractor will want to find the processed and easily
+# downloadable version. So there may be interesting bugs during the
+# race condition time before the processed video is available (which
+# is really the only time this extractor is especially important).
+
+# This is also a relatively braindead extractor. It parses a given page like
+#   http://cambridgema.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=1679
+# to determine the location of an inner div defined by a URL of the form
+#   http://cambridgema.iqm2.com/Citizens/VideoScreen.aspx?MediaID=1563&Frame=SplitView
+
+# and then simply hands that URL to the GenericIE generic extractor,
+# which matches it under the "Broaden the findall a little bit:
+# JWPlayer JS loader" (line 2372 as of 6 Oct 2016).
```

Comment:
I was a little reluctant to slow down the travis-ci runs with unnecessary tests.
I guess my listing URLs there was really more along the lines of my documentation of the purpose of the regexp, see our discussion about lines 95-100, below.
What's the standard for tests? If there are multiple forms of URLs that are handled but we anticipate them being fate-sharing (e.g. `/path/to/*.aspx` all works the same), is it appropriate to include multiple test cases?